master/internal/api_runs_intg_test.go

```diff
@@ -402,7 +403,52 @@ func setUpMultiTrialExperiments(ctx context.Context, t *testing.T, api *apiServe
 
 func TestMoveRunsMultiTrialSkip(t *testing.T) {
 	api, curUser, ctx := setupAPITest(t, nil)
-	sourceprojectID, destprojectID, runID1, runID2, _ := setUpMultiTrialExperiments(ctx, t, api, curUser)
```

Comment:
question: Is there a reason why we're are moving away from `setUpMultiTrialExperiments` here and not in the remainder of the test?

Reply:
In order to setup a non-single experiment I had to pass in a config with a non-single searcher. The functions that were setting up the experiment had to essentially be flattened out in this case in order to pass in the config. The other cases don't need the searcher type as we aren't skipping the multitrial